app/services/ResourceIndexer.java

```diff
@@ -48,12 +55,18 @@
     "    FILTER ( !BOUND(?y) ) " +
     "}";
 
-  public ResourceIndexer(Model aDb, Writable aTargetRepo, GraphHistory aGraphHistory) {
-
-    this.mDb = aDb;
-    this.mTargetRepo = aTargetRepo;
-    this.mGraphHistory = aGraphHistory;
 
+  public ResourceIndexer(Model aDb, Writable aTargetRepo, GraphHistory aGraphHistory, final Types aTypes) {
```

Comment:
Why do we need to pass `Types` to the ResourceIndexer? That parameter does not seem to be used there.

Reply:
It's needed as a parameter for the ResourceFramer to be constructed within this constructor. Thanks for pointing me here anyway, I've added a logging message.

Reply:
Oh yes... it's a relic. Sorry for that.